07.ls_object/ls.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'optparse'
+require_relative 'longformat'
+require_relative 'shortformat'
+
+class Ls
+  def initialize(options)
+    @options = options
+    @filenames = create_filenames
+  end
+
+  def list_files
+    @options['l'] ? LongFormat.new(@filenames).list_files : ShortFormat.new(@filenames).list_files
```

Comment:
条件分岐はできるだけ局所的にすると、見やすいです

```suggestion
    format_class = @options['l'] ? LongFormat : ShortFormat
    format_class.new(filenames).list_files
```

Reply:
わあ！こんなに読みやすさが違うんですね。
教えてくださってありがとうございます🙏

[7859907](https://github.com/smallmonkeykey/ruby-practices/pull/11/commits/78599070a047a384896dc96e46146bf6795fb125)